### PR TITLE
openssh: add feedstock

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: recipes/openssh/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - type -f sshd
+    - type -f ssh-keygen
+    - type -f ssh-keyscan
+
+about:
+  home: https://www.openssh.com/
+  license: BSD-style
+  license_family: BSD
+  license_file: LICENCE
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
```
Comment (bjodah) [inline recipes/openssh/meta.yaml]: I think this is from the example recipe? :)

File: recipes/openssh/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package:
+  name: openssh
+  version: 7.9p1
+
+source:
+  url: http://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-7.9p1.tar.gz
```
Comment (chrisburr) [inline recipes/openssh/meta.yaml]:
```suggestion
  url: http://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/{{ name }}-{{ version }}.tar.gz
```

File: recipes/openssh/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+package:
```
Comment (chrisburr) [inline recipes/openssh/meta.yaml]:
```suggestion
{% set name = "openssh" %}
{% set version = "7.9p1" %}

package:
```

File: recipes/openssh/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  name: openssh
```
Comment (chrisburr) [inline recipes/openssh/meta.yaml]:
```suggestion
  name: {{ name }}
```

File: recipes/openssh/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  version: 7.9p1
```
Comment (chrisburr) [inline recipes/openssh/meta.yaml]:
```suggestion
  version: {{ version }}
```

File: recipes/openssh/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    - pkg-config
+    - {{ compiler('c') }}
+  host:
+    - zlib
+    - libedit
+    - krb5
+    - openssl
+
+test:
+  commands:
```
Comment (chrisburr) [inline recipes/openssh/meta.yaml]: Maybe add tests which use some of these commands such as:
```suggestion
  commands:
    - ssh-keygen -t rsa -b 4096 -f id_rsa -P do-not-pass-passwords
```

File: recipes/openssh/build.sh [hunk 7]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+if [[ "$target_platform" == "linux-64" ]]; then
+  export LDFLAGS="-Wl,-rpath=$PREFIX/lib"
```
Comment (isuruf) [inline recipes/openssh/build.sh]: Isn't this set by the compiler?

**Pull request merged**